concepts/nullability/about.md

```diff
@@ -0,0 +1,106 @@
+# About Nullability
+
+In an ideal world, we could rely on everything being nicely defined, with concrete values and no gaps.
+
+This conflicts with our everyday experience, and thus many programming languages have a way to represent the absence of a value: null or NULL, nil, None, nothing...
+
+This was easy to introduce (back in 1965), but soon led to regrets.
+The inventor later called it his ["Billion Dollar Mistake"][web-mistake].
+
+In recent decades, language designers have explored many ways to make nulls available but (mostly) safe.
+
+Kotlin has a few [general principles][ref-null-safety], which [differ significantly from Java][ref-java2kotlin].
+_The differences mean that Kotlin has special functionality to deal with Java interop, though details are beyond the scope of this Concept._
+
+- Kotlin has a special `null` value.
+- All types _default_ to being non-nullable.
+- Most types can be _made_ nullable, but the programmer must explicitly order this.
+- Functions which might fail with an exception are likely to have a variant with a nullable return value.
+- Special, highly terse syntax tries to make testing for and responding to nulls as easy as possible.
+
+## Creating nullable variables
+
+Standard types generally have a [nullable equivalent][ref-nullable-type], with a `?` suffix: for example `String?` instead of `String`.
```

Comment:
I read "Standard types" as the types that come with Kotlin, but I think this applies to _any_ type.

```suggestion
A type can be made [nullable][ref-nullable-type] by adding the `?` suffix (e.g. `String?` instead of `String`).
```